values.yaml

```diff
@@ -235,12 +235,14 @@ livenessProbe:
   httpGet:
     path: /__health
     port: http
+    scheme: HTTPS
 
 # -- (object) The readinessProbe to use for the krakend pod
 readinessProbe:
   httpGet:
     path: /__health
     port: http
+    scheme: HTTPS
```

Comment:
We might need to use a macro for this instead of setting it up directly here. HTTPS won't work in default settings where TLS is not handled by krakend.

Reply:
Note this is still a blocker for merging this.

Reply:
Sorry, I thought I created the pull request in DRAFT mode, I wasn't done yet and I'm aware that there are some things that are not yet general enough but just take care of my own issue. I will try and generalize and address your comments